volatility/framework/plugins/windows/crashinfo.py

```diff
@@ -0,0 +1,42 @@
+  
+# This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from volatility.framework import interfaces, renderers
```

Comment:
The namespace was very recently changed from `volatility` to `volatility3`.  We took the opportunity to reset the framwork verison as well, so `_required_framework_version` should be set to `(1, 0, 0)`.